lib/neek.js

```diff
@@ -56,14 +56,20 @@ module.exports = {
 
 };
 
-function isReadStream(rs){
-  return isStream(rs, process.stdin, '_read');
+function isReadStream(obj) {
+  // some streams are readable but do not have `read()`, however
+  // we can reliably detect readability by testing for `resume()`
+  return isStream(obj) && typeof obj.resume === 'function';
```

Comment:
Not sure I'm clear on this. According to the docs, `read()` has been around since v0.9: https://nodejs.org/api/stream.html#stream_readable_read_size?